### PR TITLE
[e-speak-fe-2] Initial(UI): record 

### PR DESCRIPTION
**Pull request by khanh181**
##### Make sure all of them are checked before assigning reviewers

- [x] There's no conflict with the main branch
- [x] All checks has been successfully passed
- [x] There is no warnings in the console
- [x] I have checked the task satisfies what is asked in project
- [x] I did the QA part and I think there are no bugs
- [x] The title's format is correct

### CODE REVIEW

File: src/app/[lang]/(home)/page.tsx [hunk 1]
```diff
@@ -10,10 +10,10 @@ import 'slick-carousel/slick/slick-theme.css'
...
-  console.log(dictionary); // eslint-disable-line
-  console.log('%c => dictionary ', 'background: #0095FF; color: #fff'); // eslint-disable-line
-  console.log(new Date()); // eslint-disable-line
+  console.log(`🆘 src/app/[lang]/(home)/page.tsx`); // eslint-disable-line
+  console.log(dictionary); // eslint-disable-line
+  console.log('%c => dictionary ', 'background: #0095FF; color: #fff'); // eslint-disable-line
+  console.log(new Date()); // eslint-disable-line
 
-  return <Homepage />
+  return <AudioPage />
```
Comment (tranduybau) [inline src/app/[lang]/(home)/page.tsx:18]: It's still <homepage />. AudioWrapper or Audio (component)

File: src/components/business/record-page/audioRecord.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+'use client'
```
Comment (tranduybau) [inline src/components/business/record-page/audioRecord.tsx:1]: `src/components/business/homepage/index.tsx`. Inside we have component Audio

File: src/components/business/record-page/audioRecord.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        <div className="mb-4 flex space-x-4">
+          <Button
+            onClick={isRecording ? stopRecording : startRecording}
+            className={cn(
+              'text-white',
+              isRecording ? 'bg-red-500 hover:bg-red-600' : 'bg-blue-500 hover:bg-blue-600',
+            )}
+          >
+            {isRecording ? <Square className="mr-2" /> : <Mic className="mr-2" />}
+            {isRecording ? 'Stop Recording' : 'Start Recording'}
```
Comment (tranduybau) [inline src/components/business/record-page/audioRecord.tsx:79]: Please translate it

File: src/dictionaries/en.json [hunk 4]
```diff
@@ -1,3 +1,74 @@
-{
-  "hello": "hello"
-}
+
+{
```
Comment (tranduybau) [inline src/dictionaries/en.json:2]: what is this?

Comment (khanh181) [reply src/dictionaries/en.json:2]: had resolve in another pr

File: .prettierrc [hunk 5]
```diff
@@ -1,3 +1,6 @@
-{
-  "plugins": ["prettier-plugin-tailwindcss"]
-}
+{
+  "plugins": ["prettier-plugin-tailwindcss"],
+  "prettier/prettier": ["error", {
+      "endOfLine": "auto" }
+  ]
+}
```
Comment (tranduybau) [inline .prettierrc:6]: Revert code please

File: src/app/[lang]/(home)/page.tsx [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 
-import Homepage from '@/components/business/home-page'
+// import AudioComponent from '@/components/business/recordpage'
```
Comment (tranduybau) [inline src/app/[lang]/(home)/page.tsx:3]: Remove if you dont use it

File: src/app/[lang]/(home)/page.tsx [hunk 7]
```diff
@@ -1,6 +1,7 @@
...
+import Homepage from '@/components/business/homepage'
```
Comment (tranduybau) [inline src/app/[lang]/(home)/page.tsx:4]: Why change from `home-page` to `homepage`?

File: src/components/business/recordpage/audioRecord.tsx [hunk 8]
```diff
@@ -0,0 +1,101 @@
+'use client'
```
Comment (tranduybau) [inline src/components/business/recordpage/audioRecord.tsx:1]: Do you use this file? Wrong way to set name of folders, files. Please check it carefully

File: src/components/business/recordpage/audioRecord.tsx [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+  const stopRecording = () => {
+    if (mediaRecorder.current && isRecording) {
+      mediaRecorder.current.stop()
+      setIsRecording(false)
+    }
+  }
+
+  const downloadRecording = () => {
+    if (audioURL) {
+      const a = document.createElement('a')
```
Comment (tranduybau) [inline src/components/business/recordpage/audioRecord.tsx:56]: Why we have to create a new a element here?

File: src/dictionaries/en.json (hunk 4 above)
Comment (tranduybau) [reply src/dictionaries/en.json:2]: No, please dont change if you dont touch the file. Remove the changed please

File: src/components/business/record-page/audio-recorder.tsx [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+        <div className="mb-4 flex space-x-4">
+          <Button
+            onClick={isRecording ? stopRecording : startRecording}
+            className={cn(
+              'text-white',
+              isRecording ? 'bg-red-500 hover:bg-red-600' : 'bg-blue-500 hover:bg-blue-600',
+            )}
+          >
+            {isRecording ? <Square className="mr-2" /> : <Mic className="mr-2" />}
+            {isRecording ? dictionary['Stop Recording'] : dictionary['Start Recording']}
```
Comment (tranduybau) [inline src/components/business/record-page/audio-recorder.tsx:79]:
```suggestion
           <span> {isRecording ? dictionary['Stop Recording'] : dictionary['Start Recording']}</span>
```

File: src/components/business/record-page/audio-recorder.tsx [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+            onClick={downloadRecording}
+            disabled={!audioURL}
+            className="bg-green-500 text-white hover:bg-green-600"
+          >
+            <Download className="mr-2" />
+            <span>{dictionary['Download Recording']}</span>
+          </Button>
+        </div>
+        {audioURL && (
+          /* eslint-disable-next-line jsx-a11y/media-has-caption */
```
Comment (tranduybau) [inline src/components/business/record-page/audio-recorder.tsx:91]: Fix eslint, not disable

File: .prettierrc [hunk 12]
```diff
@@ -1,3 +1,3 @@
-{
```
Comment (tranduybau) [inline .prettierrc:1]: What the changes here? If nothing changes, it shouldn't be showed here

File: src/components/business/record-page/audio-recorder.tsx [hunk 13]
```diff
@@ -0,0 +1,85 @@
+'use client'
```
Comment (tranduybau) [inline src/components/business/record-page/audio-recorder.tsx:1]: `src/components/business/audio-recorder/index.tsx`

File: src/components/business/record-page/index.tsx [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+  const dictionary = await getDictionary(LocaleEnum.EN)
+
+  return (
+    <div className="flex min-h-screen flex-col items-center justify-center bg-gray-100">
+      <AudioRecorder dictionary={dictionary} />
+    </div>
+  )
+}
+
+export default RecordPage
```
Comment (tranduybau) [inline src/components/business/record-page/index.tsx:18]: Did you register `Record` Page in routes? Did you register in `/app`?

File: src/components/features/dowload-record.tsx [hunk 15]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
```
Comment (tranduybau) [inline src/components/features/dowload-record.tsx:1]: 1. 'use client'
2. `src/components/features/dowload-record/index.tsx`

File: src/components/business/record/index.tsx [hunk 16]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import { getDictionary } from '@/dictionaries/get-dictionary'
+import { LocaleEnum } from '@/types/locales'
+
+import AudioRecorder from './audio-recorder'
+
+async function RecordComponent() {
```
Comment (tranduybau) [inline src/components/business/record/index.tsx:8]:
```suggestion
async function RecordWrapper() {
```

**Pull request merged**